lib/oembed/scribehow_fallback_provider.rb

```diff
@@ -5,7 +5,11 @@ def self.domains
     end
 
     def self.paths
-      [%r{^/embed/.*}]
+      [%r{^/embed/.*}, %r{^/shared/.*}]
+    end
+
+    def resource_url
+      @resource_url.gsub("/shared", "/embed")
```

Comment:
It'll be safer to include the domain name in the `gsub` - otherwise it's possible that you'll end up editing the next segment of the URL, which also contains words from the name of the _Scribe_.

 ```suggestion
      @resource_url.gsub("scribehow.com/shared", "scribehow.com/embed")
```

Reply:
@harigopal 

I don't think someone will name 🤞 the title, including 'slash' in the word  `shared`  (/shared) but for being safe will have the domain as well 👍